### PR TITLE
[enterprise-4.13] ReleaseNote for AWS Local Zones - Phase 1 IPI edge pool

### PR DESCRIPTION
**Pull request by maxwelldb**
Version(s): 4.13

Issue: [OSDOCS-5575](https://issues.redhat.com//browse/OSDOCS-5575)

Link to docs preview: https://ubiquitous-snickerdoodle-96fa29.netlify.app/release_notes/ocp-4-13-release-notes.html#ocp-4-13-installation-aws-local-zones

QE review:
- [x] QE has approved this change.

RN related to https://github.com/openshift/openshift-docs/pull/57427

### CODE REVIEW

**Comment by yunjiang29** [comment]
@maxwelldb thanks for the updates, the description looks good to me 👍

cc @mtulio

**Review by JoeAldinger — COMMENTED** [review]
/lgtm ping me when you insert the xref and I'll re-look at it and merge when ready.

**Comment by maxwelldb** [comment]
@JoeAldinger Link added. Built fine for me locally.

File: release_notes/ocp-4-13-release-notes.adoc [hunk 1]
```diff
@@ -158,6 +158,13 @@ In {product-title} {product-version}, you can use Confidential VMs when installi
 Due to a known issue, you cannot use persistent volume storage on a cluster with Confidential VMs. For more information, see link:https://issues.redhat.com/browse/OCPBUGS-7582[OCPBUGS-7582].
 ====
 
+[id="ocp-4-13-installation-aws-local-zones"]
+==== Installing a cluster on AWS into an existing Virtual Private Cloud (VPC) improvements
+
+In {product-title} {product-version}, the installation process for clusters that use AWS VPCs is simplified. This release also introduces the _edge pool_, a pool of machines that are optimized for AWS Local Zones. 
+
+For more informsation, see xref:../installing/installing_aws/installing-aws-localzone.adoc#installing-aws-localzone[Installing a cluster using AWS Local Zones].
```
Comment (mtulio) [inline release_notes/ocp-4-13-release-notes.adoc:166]: @maxwelldb 
s/informsation/information/

Comment (maxwelldb) [reply release_notes/ocp-4-13-release-notes.adoc:166]: Thanks! Translators should catch this in what we sent off on Thursday. I'll make an English change tomorrow.

**Pull request merged**